multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/zeistmanga/ZeistMangaGenerator.kt

```diff
@@ -9,10 +9,10 @@ class ZeistMangaGenerator : ThemeSourceGenerator {
 
     override val themeClass = "ZeistManga"
 
-    override val baseVersionCode: Int = 2
+    override val baseVersionCode: Int = 3
 
     override val sources = listOf(
-        SingleLang("DatGarScanlation", "https://datgarscanlation.blogspot.com", "es"),
+        SingleLang("DatGarScanlation", "https://datgarscanlation.blogspot.com", "es", overrideVersionCode = 1),
```

Comment:
Don't need to increase version here when you increase the base version in the same PR
```suggestion
        SingleLang("DatGarScanlation", "https://datgarscanlation.blogspot.com", "es"),
```